setup.cfg

```diff
@@ -24,6 +24,7 @@ install_requires =
     awscli
     spec-plots==1.34.6
     psutil
+    voila<0.2
```

Comment:
what's this for?

Reply:
Good catch.   I misread a pip error which really shouldn't have been happening either.   (Pip reported broken dependencies from some past install which did not report them...  while I was installing caldp.  I thought it was odd.)

Reply:
Should be gone now.